doc/source/getting_started/about_odl.rst

```diff
@@ -16,182 +16,6 @@ The main advantages of this approach is that
 3. Solvers and application-specific code need to be written only once, in one place, and can be tested individually.
 4. Adding new applications or solution methods becomes a much easier task.
 
-ODL implements many abstract mathematical notions such as sets, vector spaces and operators.
-In the following, a few are shown by example.
```

Comment:
All the stuff below seemed utterly pointless to me. What normal user cares about this stuff?

Reply:
I agree, go ahead.